### ISSUE TITLE
Why was lv_drivers removed in version V9.0?

### ISSUE THREAD

**Issue by chenzhujin** [lvgl/lv_port_linux#56]
I am unable to use the mouse and touch screen in version V9.0 on the Ubuntu system and development board because the lv_drivers module is missing. How can I add this module to the V9.0 package?

**Comment by chenzhujin** [comment]
int main(void)
{
    lv_init();
    /*Linux display device init*/
    lv_linux_disp_init();

    printf("init input\n");

    //init input device
    lv_sdl_mouse_create();
    lv_sdl_keyboard_create();
    lv_sdl_mousewheel_create();
    lv_sdl_mousewheel_create();

    printf("init input  end\n");

    /*Create a Demo*/
   // lv_example_button_1();
    // lv_demo_widgets();
    // lv_demo_widgets_start_slideshow();
     lv_example_ime_pinyin_2();
    /*Handle LVGL tasks*/
    while(1) {
        lv_timer_handler();
        usleep(5000);
    }
    return 0;
}

I know that V9.0 has integrated drivers into the src directory, and you can use them after adding initialization in the project. Thank you！

**Comment by YobeZhou** [comment]
> I know that V9.0 has integrated drivers into the src directory, and you can use them after adding initialization in the project. 

Yes, they are here: [https://github.com/lvgl/lvgl/tree/release/v9.0/src/drivers](https://github.com/lvgl/lvgl/tree/release/v9.0/src/drivers)

**Comment by AndreCostaaa** [comment]
Hi, i'm glad you were able to find the new drivers' location. Closing this now, thank you :)
